lib/api/web-element/commands/getText.js

```diff
@@ -24,6 +24,7 @@
  * @syntax browser.element(selector).getText()
  * @see https://www.w3.org/TR/webdriver#dfn-get-element-text
  * @returns {ScopedValue<string>}
+ * @alias text
```

Comment:
These aliases are only for backward compatibility, we don't need to specify these in the JSDoc.

Reply:
Same for all PRs.